packages/documentation-framework/components/themeSelector/themeSelector.js

```diff
@@ -45,6 +50,41 @@ const DesktopIcon = (
   </svg>
 );
 
+const HCGroupLabel = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  return (
+    <h2 className="pf-v6-c-menu__group-title">
+      High contrast{' '}
+      <Popover
```

Comment:
This may depend on discussion of my PR comment above, but if we do decide to keep heading levels within this menu, could we structure this in a way so that the popover button and beta label aren't part of the heading? This is what the heading ends up looking like in the a11y tree:

<img width="552" height="203" alt="image" src="https://github.com/user-attachments/assets/9c80ae36-3540-4085-9335-28593ac03140" />

Which feels odd for some reason. I think if it was just the High Contrast text and the Beta label it'd look a little better.